src/sdk/config_fs.js

```diff
@@ -9,12 +9,37 @@ const nb_native = require('../util/nb_native');
 const native_fs_utils = require('../util/native_fs_utils');
 const nc_mkm = require('../manage_nsfs/nc_master_key_manager').get_instance();
 
+/* Config directory sub directory comments - 
+   On 5.18 - 
+   1. accounts/ will be deprecated
+   2. A new identities/ directory will be created that represents an account identity, 
+      for example -
+      when an account called alice is created (id = 111) and an IAM user called bob (id = 222) 
+      was created for alice, in the FS it will look as the folloing - 
+      2.1. config_dir/identities/111/identity.json -> represents the account properties (alice)
+      2.2. config_dir/identities/111/users/ -> represents the account's iam users
+      2.3. config_dir/identities/111/users/bob.symlink -> config_dir/identities/222/identity.json (an index to another identity)
+      2.4. config_dir/identities/222/bob.json -> represents the account properties (bob)     
+      in the future the identity directory will contain more features like policies, roles etc.
+   3. A new accounts_by_name/ directory will be created that represents an index which is a 
+      a symlink from account name to its actual identity.
+      For example - 
+      config_dir/accounts_by_name/alice.symlink -> config_dir/identities/111/identity.json
+*/
+
 const CONFIG_SUBDIRS = Object.freeze({
-    ACCOUNTS: 'accounts',
     BUCKETS: 'buckets',
-    ACCESS_KEYS: 'access_keys'
+    ACCESS_KEYS: 'access_keys',
+    IDENTITIES: 'identities',
+    ACCOUNTS_BY_NAME: 'accounts_by_name',
+    ACCOUNTS: 'accounts', // deprecated on 5.18
 });
 
+const CONFIG_TYPES = {
+    ACCOUNT: 'account',
```

Comment:
It is worth adding a comment that this ACCOUNT type is for account and user config.